AppleDev.Tool/Program.cs

```diff
@@ -17,80 +17,122 @@
 var app = new CommandApp();
 app.Configure(config =>
 {
+	config.SetApplicationName("apple");
+	
+	config.AddExample(new[] { "simulator", "list" });
+	config.AddExample(new[] { "simulator", "create", "\"My iPhone 15\"", "--device-type", "\"iPhone 15\"" });
+	config.AddExample(new[] { "device", "list" });
+	config.AddExample(new[] { "provisioning", "list", "--download" });
+	config.AddExample(new[] { "ci", "provision", "--certificate", "IOS_CERT_BASE64", "--bundle-identifier", "com.myapp.id" });
```

Comment:
I added better examples as the default is to take the first 5 and that meant it was all the list ones.